Promact.OAuth.Client/src/Promact.OAuth.Client/Util/StringConstant/StringConstant.cs

```diff
@@ -428,7 +428,7 @@ public string PromactOAuthClientId
         {
             get
             {
-                return "OOKK4SVDYQ8XRUU";
+                return "PromactOAuthClientId";
```

Comment:
Test related constants should be in Test project. here you could return Environment.GetVariable